airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/conftest.py

```diff
@@ -117,7 +117,8 @@ def malformed_connector_config_fixture(connector_config) -> MutableMapping[str,
 
 @pytest.fixture(name="connector_spec")
 def connector_spec_fixture(connector_spec_path) -> ConnectorSpecification:
-    return ConnectorSpecification.parse_file(connector_spec_path)
+    spec_obj = load_yaml_or_json_path(connector_spec_path)
+    return ConnectorSpecification.parse_obj(spec_obj)
```

Comment:
looks like we can't test these fixtures directly, so moved the yaml/json logic into a utility method